build.gradle

```diff
@@ -462,28 +465,60 @@ task downloadLinuxJbr(type: Copy) {
     into(downloadDir)
 }
 
-task fixMacOsSymbolicLink(group: 'jbr') {
+task bundleMacosJbr() {
+    onlyIf {
+        !OperatingSystem.current().isWindows()
```

Comment:
We should probably add some kind of warning here saying that this doesn't happen on windows